dev/build/generate.sh

```diff
@@ -38,14 +38,14 @@ install_controller_gen() {
 }
 
 generate_deepcopy_methods_file() {
-  local cmd=("${CONTROLLER_GEN}" object:headerFile="${SCRIPT_DIR}/license-header/go.txt" paths="${SCRIPT_DIR}/../...")
+  local cmd=("${CONTROLLER_GEN}" object:headerFile="${SCRIPT_DIR}/license-header/go.txt" paths="${SCRIPT_DIR}/../../...")
   "${cmd[@]}"
 }
 
 generate_crds() {
-  local cmd=("${CONTROLLER_GEN}" crd paths="${SCRIPT_DIR}/../..." output:crd:artifacts:config="${SCRIPT_DIR}/../resources/crds")
+  local cmd=("${CONTROLLER_GEN}" crd paths="${SCRIPT_DIR}/../../..." output:crd:artifacts:config="${SCRIPT_DIR}/../../resources/crds")
```

Comment:
just wondering why does the crd paths value end in 3 `.`s instead of 2?

Reply:
"It considers every package in the current directory tree (as per the normal rules of the go ... wildcard)."
https://book.kubebuilder.io/reference/controller-gen.html